### PR TITLE
`milestone` step released

### PR DESCRIPTION
**Pull request by amuniz**
[`milestone` step 1.0](https://github.com/jenkinsci/pipeline-milestone-step-plugin/pull/1) has been released, reflecting it in the demo + plugins update.

@reviewbybees

### CODE REVIEW

**Comment by amuniz** [comment]
Mmm, investigating test failures, it seems like http-client leaked through some dependency.

File: demo/snapshot-only-plugins.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-pipeline-milestone-step
```
Comment (jglick) [inline demo/snapshot-only-plugins.txt]: BTW I intend to rewrite how plugins are selected to match the much superior system in https://github.com/jenkinsci/parallel-test-executor-plugin/pull/20.

**Comment by amuniz** [comment]
@jglick tests fixed.

**Comment by abayer** [comment]
:bee:

**Pull request merged**